### ISSUE TITLE
Error "page misalignment" in files  pg_internal.init and pg_filenode.map.

### ISSUE THREAD

**Issue by blogh** [pgbackrest/pgbackrest#327]
Hi, 

I get the error : "page misalignment in files  pg_internal.init and pg_filenode.map. For me these file have the right size.

```
postgres@ulbdybbb02:~$ pgbackrest backup --stanza=iybbt001 --type=full --process-max=4 --no-checksum | grep WARN
postgres@ulbdybbb02:~$ pgbackrest backup --stanza=iybbt001 --type=full --process-max=4 --checksum | grep WARN
WARN: page misalignment in file /home/postgres/data/iybbt001/systeme/pg_tblspc/16396/PG_9.5_201510051/16397/pg_internal.init: file size 122536 is not divisible by page size 8192
WARN: page misalignment in file /home/postgres/data/iybbt001/systeme/pg_tblspc/16396/PG_9.5_201510051/16397/pg_filenode.map: file size 512 is not divisible by page size 8192
```

This is reproductible with a tablespace outside of $PGDATA

```
postgres@ulbdybbb02:~/data/iybbt001/systeme/base/12418$ ls -al /home/postgres/data/iybbt001/systeme/pg_tblspc
total 12
drwx------  2 postgres postgres 4096 Mar  2 15:15 .
drwx------ 18 postgres postgres 4096 Mar  2 16:32 ..
lrwxrwxrwx  1 postgres postgres   67 Mar  2 15:15 16396 -> /home/postgres/data/iybbt001/tablespaces/bybb001/ts_bybb001_default
```

Benoit

Edit: mis type

**Comment by dwsteele** [comment]
Interesting - have you made this the default tablespace for the database?  It's an easy fix, though.

Thanks!

**Comment by dwsteele** [comment]
OK - reproduced it.  Will work on a fix.

Meanwhile, these are just warnings and do not affect the backup.  I know it's annoying, but the backups are still good.

**Comment by dwsteele** [comment]
Fixed in 02730526fc1a157231293ae532882847f7015a46.
